scripts/jenkins/ardana/ansible/init.yml

```diff
@@ -252,3 +266,28 @@
       chdir: /var/lib/ardana/scratch/ansible/next/ardana/ansible
     become: true
     become_user: ardana
+    register: tempest_run_result
+    ignore_errors: true
+    when: site_result|succeeded
+
+  - name: Record any failure of tempest
+    set_fact:
+      failures: "{{ failures + ['tempest'] }}"
+    when: tempest_run_result|failed
+
+  - include: post-deploy-checks.yml
```

Comment:
I am wondering if we want to run the post-deploy-checks after site.yml, not after tempest. If the playbooks that install tempest are doing something ugly, maybe is not so wrong.